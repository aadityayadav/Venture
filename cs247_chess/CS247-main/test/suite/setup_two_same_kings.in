setup
+ k e4
+ k e5
done
- e5
+ K e8
done
quit
