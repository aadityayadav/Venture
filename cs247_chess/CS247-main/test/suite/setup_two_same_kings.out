The board must contain exactly one king for each colour!
____K___
________
________
________
____k___
________
________
________
